enviro/__init__.py

```diff
@@ -549,6 +549,12 @@ def sleep(time_override=None):
   rtc.set_alarm(0, minute, hour)
   rtc.enable_alarm_interrupt(True)
 
+  # always assume we're running on battery power until we know otherwise
+  if config.usb_power:
+    logging.debug("assume battery power in config")
```

Comment:
Minor thing, but any logging should be correctly indented for the logs around it, with the appropriate symbol. e.g. `logging.debug("  - assume battery power in config")`